### DISCUSSION TITLE
Can be use gleam as a nodejs library

### DISCUSSION THREAD

**Discussion by wingyplus** [gleam-lang/gleam#2297] [Questions & support] [answered]
As title said, i want to transpile to js and use it as a library but I’m not sure how to do it.

**Comment by lpil** [comment]
Hello! Yes, you could take the compiled JavaScript code and distribute it. One thing to consider though is that you'd also need to include any Gleam packages that your project depends on, so you may want to consider using a bundler to include them all.

**Reply by wingyplus** [reply]
So, the step is:

1. Setup npm package.json
2. Install bundler and setup bundler to bundle all files under `build/[dev,prod]/javascript/package_name/*.mjs`
3. Publishing them.

Am I right?

**Reply by lpil** [reply, marked as answer]
That sounds good to me!

**Reply by wingyplus** [reply]
Thank you. ^^
